fly/fly.py

```diff
@@ -8,25 +9,23 @@
 class Fly:
     def __init__(
             self, concourse_url, executable='/usr/local/bin/fly',
-            platform='darwin', target='default'
+            target='default'
     ):
         self.concourse_url = concourse_url
         self.executable = executable
-        self.platform = platform
+        self.platform = platform.system().lower()
         self.target = target
 
+    def get_fly(self):
         if not os.path.isfile(self.executable):
-            self._get_fly()
```

Comment:
Why make the user have to remember to call `get_fly`?

Reply:
If the call to `get_fly` is idempotent, it doesn't hurt to call it automatically every time.

Reply:
the problem is the webhook dispatcher is async so this is going to be annoying